src/cmd-meta

```diff
@@ -4,16 +4,43 @@ cmd-meta is a helper for interacting with a builds meta.json
 """
 
 import argparse
+import glob
 import os
 import sys
 
 COSA_PATH = os.path.dirname(os.path.abspath(__file__))
 sys.path.insert(0, COSA_PATH)
-from cosalib.meta import GenericBuildMeta as Meta
+from cosalib.cmdlib import write_json
+from cosalib.meta import (
+    GenericMeta,
+    GenericBuildMeta,
+    COSA_DELAYED_MERGE,
+    COSA_VER_STAMP
+)
+
+
+def pather(val):
```

Comment:
Can you add a docstring/comment here with an example transformation this function does? Edit: ahh, I see this isn't new code. Comment still would be nice but feel free to leave as a follow-up.